admin_manual/enterprise_clients/publishing_ios_app.rst

```diff
@@ -0,0 +1,121 @@
+===================================================
+Distributing Your Branded iOS App (Enterprise Only)
+===================================================
+
+########## **Please See Questions At the End** ###########
+
+Creating and distributing your branded iOS ownCloud app involves a number of 
+interdependent steps. First you must create three provisioning profiles, then 
+use ownBrander to create your branded app, and then email your provisioning 
+profiles to branding@owncloud.com.
+
+Prerequisites
+-------------
+
+* A Mac OS X computer with Xcode (free download) and Keychain Access (Utilities)
+* An account on developer.apple.com
+* An ownCloud Enterprise Subscription & the ownBrander app on 
+  customer.owncloud.com
+  
+Procedure
+---------
+
+1. Create a digital signing infrastructure on developer.apple.com. Apple 
+   requires that all iOS apps are digitally signed.
+
+2. Create three provisioning profiles.
+
```

Comment:
If they go for the developer account, before creating the provisioning profiles, they will have to register any device UDID (without them, it won't work)